core/src/test/scala/unit/kafka/log/LogTest.scala

```diff
@@ -2634,17 +2634,16 @@ class LogTest {
     val downgradedLogConfig = LogTest.createLogConfig(segmentBytes = 1000, indexIntervalBytes = 1,
       maxMessageBytes = 64 * 1024, messageFormatVersion = kafka.api.KAFKA_0_10_2_IV0.shortVersion)
     log.updateConfig(Set(LogConfig.MessageFormatVersionProp), downgradedLogConfig)
-    assertLeaderEpochCacheEmpty(log, 5)
+    assertLeaderEpochCacheEmpty(log)
 
     log.appendAsLeader(TestUtils.records(List(new SimpleRecord("bar".getBytes())),
       magicValue = RecordVersion.V1.value), leaderEpoch = 5)
-    assertLeaderEpochCacheEmpty(log, 5)
+    assertLeaderEpochCacheEmpty(log)
   }
 
-  private def assertLeaderEpochCacheEmpty(log: Log, epoch: Int): Unit = {
-    val (leaderEpoch, offset) = log.leaderEpochCache.endOffsetFor(epoch)
-    assertEquals(UNDEFINED_EPOCH, leaderEpoch)
-    assertEquals(UNDEFINED_EPOCH_OFFSET, offset)
+  private def assertLeaderEpochCacheEmpty(log: Log): Unit = {
+    assertEquals(UNDEFINED_EPOCH, log.leaderEpochCache.latestEpoch)
```

Comment:
nit: below we use `cache.epochEntries.isEmpty`. Is there any reason not to make the checks consistent?